lxd/instance_state.go

```diff
@@ -146,3 +152,14 @@ func doInstanceStatePut(inst instance.Instance, req api.InstanceStatePut) error
 
 	return fmt.Errorf("Unknown action: '%s'", req.Action)
 }
+
+func createSnapshotIfScheduled(inst instance.Instance, d *Daemon, timeout int) error {
```

Comment:
Based on my comment above, this function likely will not be needed anymore.
If a function like it is still needed, it may be better suited for the `instance` package?